src/include/execution/sql/runtime_types.h

```diff
@@ -646,3 +651,115 @@ inline Timestamp Date::ConvertToTimestamp() const { return Timestamp(value_ * K_
 inline Date Timestamp::ConvertToDate() const { return Date(value_ / K_MICRO_SECONDS_PER_DAY); }
 
 }  // namespace noisepage::execution::sql
+
+namespace std {
```

Comment:
Always a sweet day when you crack open `std`.

Reply:
Is there a better way of doing this? I wasn't really sure.

Reply:
Nope as far as I am aware this is relatively-common practice to overload `std::hash` for our own types. Resolved.